### PR TITLE
Using regex

### PR DESCRIPTION
**Pull request by nouroddinc**
please check this new version and add slims to it. thanks

### CODE REVIEW

File: summary.py [hunk 1]
```diff
@@ -400,149 +359,6 @@ def chromap_log_stats(file_path, target_string):
 summary_df.to_csv("/root/Statistics/summary.csv", header=True, index= False)
 print(f"Output written to summary.csv")
 
-def get_axis_avgs(
```
Comment (jpmcga) [inline summary.py:403]: Was the plotting task moved somewhere else?

**Review by jpmcga — APPROVED** [review]
Looks good.  I added back lims and the plotting task. updeated the version to .5

**Comment by jpmcga** [comment]
@nouroddinc there are some changes between qc.py in main and summary.py.  I'm not sure which changes to keep!  Is the correct version the one in this branch?

**Comment by nouroddinc** [comment]
Yes. The right version is what is in summary.py and singlecell.py

**Pull request merged**